lib/galaxy/jobs/handler.py

```diff
@@ -125,7 +125,7 @@ def grab_unhandled_items(self):
             try:
                 rows = conn.execute(self._grab_query).fetchall()
                 if rows:
-                    log.debug('Grabbed %s(s): %s', self.grab_type, ', '.join(str(row[0]) for row in rows))
+                    log.debug(f"Handler {self.app.config.server_name} with PID {os.getpid()} grabbed {self.grab_type}(s): {', '.join(str(row[0]) for row in rows)}")
```

Comment:
```suggestion
                    log.debug(f"Handler {self.app.config.server_name} in thread {threading.get_ident()} with PID {os.getpid()} grabbed {self.grab_type}(s): {', '.join(str(row[0]) for row in rows)}")
```
need to import threading then as well

Reply:
As far as I can tell we only start a single thread though, so maybe that's not necessary ...

Reply:
@mvdbeek I think you're right that there's only one thread doing this bit, so it may not be useful.  Totally up to you though, happy to add it if you want.  I just had `handler_name` at first, then added `pid` to identify (potentially) when we have two handlers with the same name.

Reply:
Yeah, this is fine and should help already.